tf-modules/cumulus_ecs_service/variables.tf

```diff
@@ -99,3 +99,15 @@ variable "log_destination_arn" {
   default     = null
   description = "A shared AWS:Log:Destination that receives logs in log_groups"
 }
+
+variable "health_check" {
+  description = "Health check used by AWS ECS to determine containers health status. See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_healthcheck"
+  type = object({
+    command = list(string)
+    interval = number
+    timeout = number
+    retries = number
+    startPeriod = number
+  })
+  default = null
```

Comment:
Something to note for the future is that, when we upgrade terraform, we will be able to set 
 ```
     interval = number
    timeout = number
    retries = number
    startPeriod = number
   ```
   
  to 
  ```
      interval = optional(number)
    timeout =  optional(number)
    retries =  optional(number)
    startPeriod =  optional(number)
```

since these are technically not required to set a health check. For now, users will need to set all those to null but that is fine.